etc/ssl_dist.conf

```diff
@@ -5,7 +5,10 @@
 [{server,
   [{certfile, "{{ platform_etc_dir }}/certs/cert.pem"},
    {keyfile, "{{ platform_etc_dir }}/certs/key.pem"},
-   {secure_renegotiate, true},
-   {depth, 0}]},
+   {cacertfile, "{{ platform_etc_dir }}/certs/cacert.pem"},
```

Comment:
correct me if I am wrong, if verify is `verify_none`, cacertfile won't be necessary.

Reply:
yes